ledger-tool/src/bigtable.rs

```diff
@@ -286,6 +286,13 @@ impl BigTableSubCommand for App<'_, '_> {
                                     .required(true)
                                     .help("Slots to delete"),
                                 )
+                            .arg(
+                                Arg::with_name("extract_memo")
+                                    .long("extract-memo")
```

Comment:
maybe, saner default value could be "`true`". or  make this is required to force ops to think hard for this.